src/connector/mysql.rs

```diff
@@ -185,27 +185,24 @@ impl MysqlUrl {
     }
 
     pub(crate) fn to_opts_builder(&self) -> my::OptsBuilder {
-        let mut config = my::OptsBuilder::new();
-
-        config.user(Some(self.username()));
-        config.pass(self.password());
-        config.db_name(Some(self.dbname()));
+        let mut config = my::OptsBuilder::default().user(Some(self.username()))
+        .pass(self.password())
+        .db_name(Some(self.dbname()));
```

Comment:
note: run `cargo fmt` manually. I think it doesn't work automatically on my setup because of the feature flags.